public/interactive-examples/wm2-count-revenue/index.html

```diff
@@ -3,13 +3,12 @@
   <link rel="monetization" href="https://wallet.example.com/alice" />
 
   <script src="./wm-previewer.js"></script>
-  <script type="application/javascript" src="./counter.js"></script>
+  <script defer type="application/javascript" src="./counter.js"></script>
 </head>
 
 <body>
   <p>
     Thanks for contributing
-    <span id="total">0.00 USD</span>
-    <span id="currency"></span>
+    <span id="contribution">maybe in the future</span>.
```

Comment:
@melissahenderson suggestions for a better default text? It says: "Thanks for contributing maybe in future" here.